source/jormungandr/jormungandr/scenarios/experimental.py

```diff
@@ -575,192 +42,124 @@ def __init__(self):
+                                                             request=request)
+
+        dest_fallback_durations_pool = FallbackDurationsPool(instance=instance,
+                                                             requested_place_obj=requested_dest_obj,
+                                                             modes=requested_arr_modes,
+                                                             proximities_by_crowfly_pool=dest_proximities_by_crowfly,
+                                                             places_free_access=dest_places_free_access,
+                                                             streetnetwork_path_pool=None,
+                                                             request=request)
+
+        pt_journey_pool = PtJourneyPool(instance=instance,
+                                        requested_orig_obj=requested_orig_obj,
+                                        requested_dest_obj=requested_dest_obj,
+                                        streetnetwork_path_pool=streetnetwork_path_pool,
+                                        krakens_call=krakens_call,
+                                        orig_fallback_durations_pool=orig_fallback_durations_pool,
+                                        dest_fallback_durations_pool=dest_fallback_durations_pool,
+                                        request=request)
+
+        completed_pt_journeys = wait_and_complete_pt_journey(requested_orig_obj=requested_orig_obj,
+                                                             requested_dest_obj=requested_dest_obj,
+                                                             pt_journey_pool=pt_journey_pool,
+                                                             streetnetwork_path_pool=streetnetwork_path_pool,
+                                                             orig_places_free_access=orig_places_free_access,
+                                                             dest_places_free_access=dest_places_free_access,
+                                                             orig_fallback_durations_pool=orig_fallback_durations_pool,
+                                                             dest_fallback_durations_pool=dest_fallback_durations_pool,
+                                                             request=request)
+
+        for mode in requested_dep_modes:
+            dp = streetnetwork_path_pool.wait_and_get(requested_orig_obj=requested_orig_obj,
+                                                      requested_dest_obj=requested_dest_obj,
+                                                      mode=mode,
+                                                      period_extremity=period_extremity,
+                                                      streetnetwork_path_type=StreetNetworkPathType.DIRECT)
+            if getattr(dp, "journeys", None):
+                res.append(dp)
+
+        res.extend([j for j in completed_pt_journeys if j])
```

Comment:
Nope, it broke some integration tests, `res` must be a list of `protobuf journey`